sonar-duplications/src/main/java/net/sourceforge/pmd/cpd/Tokenizer.java

```diff
@@ -45,7 +45,9 @@
  * in this case the two files will be considered as duplicate.
  * 
  * @since 2.2
+ * @deprecated since 5.5
  */
+@Deprecated
 public interface Tokenizer {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)